OpenDreamRuntime/Procs/DMOpcodeHandlers.cs

```diff
@@ -744,12 +744,7 @@ private static void HandleSuffixPronoun(ref StringBuilder formattedString, ReadO
             DMReference reference = state.ReadReference();
             DreamValue value = state.GetReferenceValue(reference, peek: true);
 
-            if (value.TryGetValueAsInteger(out int intValue)) {
-                state.AssignReference(reference, new(intValue + 1));
-            } else {
-                //If it's not a number, it turns into 1
-                state.AssignReference(reference, new(1));
-            }
+            state.AssignReference(reference, new(value.UnsafeGetValueAsInteger() + 1));
```

Comment:
Keep the comment, it's useful context
```suggestion
            //If it's not a number, it turns into 1
            state.AssignReference(reference, new(value.UnsafeGetValueAsInteger() + 1));
```